app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split(" ")
+    command = command_parts[0]
+
+    if command != "cp":
+        return
+
+    source_file = command_parts[1]
+    destination_file = command_parts[2]
+
+    if source_file == destination_file:
+        return
```

Comment:
You can use 1 if for return:
if source_file == destination_file or command != "cp":
    return